src/Cubical/IsoToEquiv.agda

```diff
@@ -0,0 +1,80 @@
+{-# OPTIONS --cubical --postfix-projections #-}
+module Cubical.IsoToEquiv where
+
+open import Cubical hiding (fill)
+open import Cubical.Comp
+open import Cubical.FromStdLib
+open import Cubical.NType.Properties
+
+Square : ∀ {ℓ} {A : Set ℓ} {a0 a1 b0 b1 : A}
+          (u : a0 ≡ a1) (v : b0 ≡ b1) (r0 : a0 ≡ b0) (r1 : a1 ≡ b1) → Set ℓ
+Square {A = A} u v r0 r1 = PathP (λ i → (u i ≡ v i)) r0 r1
+
+module _ {ℓ ℓ'} {A : Set ℓ} {B : Set ℓ'} (f : A → B) (g : B → A)
+         (s : (y : B) → (f (g y)) ≡ y) (t : (x : A) → (g (f x)) ≡ x) where
+
+  private
+    module _ (y : B) (x0 x1 : A) (p0 : y ≡ (f x0)) (p1 : y ≡ (f x1)) where
+
+      fill0 : ∀ (_ _ : I) → A
+      fill0 i j = fill (λ _ → A) _ (λ k → λ { (i = i1) → t x0 k
+                                            ; (i = i0) → g y })
+                                   (inc (g (p0 i))) j
+
+      fill1 : ∀ (_ _ : I) → A
+      fill1 i j = fill (λ _ → A) _ (λ k → λ { (i = i1) → t x1 k
+                                            ; (i = i0) → g y })
+                                   (inc (g (p1 i))) j
+
+      fill2 : ∀ (_ _ : I) → A
+      fill2 i j = fill (λ _ → A) _ (λ k → λ { (i = i1) → fill1 k i1
+                                            ; (i = i0) → fill0 k i1 })
+                                   (inc (g y)) j
+      p : x0 ≡ x1
+      p i = fill2 i i1
+
+      sq  : ∀ (_ _ : I) → A
+      sq i j = comp (λ _ → A) (λ k → λ { (i = i1) → fill1 j (~ k)
+                                       ; (i = i0) → fill0 j (~ k)
+                                       ; (j = i1) → t (fill2 i i1) (~ k)
+                                       ; (j = i0) → g y })
+                              (inc (fill2 i j))
+
+      sq1 : Square {A = B} (λ _ → y) (λ i → f (p i)) p0 p1
+      sq1 i j = comp (λ _ → B) (λ k → λ { (i = i1) → s (p1 j) k
+                                        ; (i = i0) → s (p0 j) k
+                                        ; (j = i1) → s (f (p i)) k
+                                        ; (j = i0) → s y k })
+                               (inc (f (sq i j)))
+
```

Comment:
Could we change the definition of `comp` and `fill` so that they always do `inc` on the last argument (the base of the box)?